### DISCUSSION TITLE
Project Starters

### DISCUSSION THREAD

**Discussion by directormac** [daedalus-developers/.github#10] [General]
Hi Everyone as i discussed earlier on the group chat i am inviting everyone to collaborate and build starter templates for any given stack that we ourselves can utilize

A good base documentation like [this](https://realworld-docs.netlify.app/docs/implementation-creation/introduction) would suffice,
to build combinations of frameworks! could be a react+php react+laravel react+remix react+express react+fastify many more
a good example of implementaions from the "RealWorld" implementations are [here](https://codebase.show/projects/realworld?category=fullstack) , 
![image](https://github.com/daedalus-developers/.github/assets/5866196/ef0279e3-4ced-4e78-b5a8-0aa970a038c4)

Please suggest some specifications that we can build upon.

**Comment by directormac** [comment]
@gestanestle will represent the java world.

**Comment by gestanestle** [comment]
I plan to build some for spring ecosystem (integration w/different technologies and dependencies), and some for react/nextjs w/ different UI libraries. How/where do we place them here?

**Comment by Frostzeichen** [comment]
Should I go for something like Drizzle / pg-node for Postgres that can run on either Vercel or Docker? I'm working on integrating Drizzle to my article writer project rn, might as well make a guide about it too while I'm at it

Also, is this going to be strictly web-based? Seems we've got a couple of folks working with Arduino for their school projects. Might be helpful to just show them the boilerplate whenever we get someone working on MCUs and WiFi modules. I could try modifying my code for the old ESP32 (C++/Arduino) and RPi Pico servers (MicroPython) and add them here. That is, _IF_ I could find them from my backups.

I'll need to modernize them first though, since Arduino released a new core sometime last week. That could mean a big change for ESP32-based systems. So the latter might take a while to do. I can start working on the Drizzle boilerplate come next week once I've figured out how to deploy my writer app on Vercel.

**Comment by directormac** [comment]
> Also, is this going to be strictly web-based? 

Not entirely web based, i am currently waiting on tauri to have a stable mobile release where we can built a static template that can be deployed mobile, web and desktop on a single codebase, a todo/ contact list app would suffice for it, documenting what we are gonna build would help it grow and track why we did it and why are we doing it.



To put icing on the top should we just go on and generate a system design based on what the teams have done? What about a pageant voting system? we can build on top of the voting system that you guys have researched. 




> I plan to build some for spring ecosystem (integration w/different technologies and dependencies), and some for react/nextjs w/ different UI libraries. How/where do we place them here?

A good amount of  react components that can work on next/remix/astro would be good.
